src/pages/lessons/[slug].tsx

```diff
@@ -754,7 +752,7 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
               >
                 <TabList className="md:text-lg text-normal font-semibold bg-transparent space-x-1">
                   {transcriptAvailable && <Tab>Transcript</Tab>}
-                  <Tab>Comments</Tab>
+                  {commentsAvailable && <Tab>Comments</Tab>}
```

Comment:
this doesn't advance the feature. We need to show a comment box if they are able to comment and a CTA if they are not able to comment. If this PR included that markup, then I can wire it to the backend and advance the functionality. 

I feel like this is a regression.

Reply:
hmmm, really 🤔
let me refactor it with markup for the comment field